.github/workflows/lint.yml

```diff
@@ -13,11 +13,19 @@ jobs:
   lint-swift:
     name: Lint Swift
     runs-on: ubuntu-24.04 # "Noble Numbat"
-    container: swift:6.1-noble
     steps:
       - uses: actions/checkout@v4
+      - name: Create ci.bazelrc file
+        run: echo "$CI_BAZELRC_FILE_CONTENT" | base64 -d > ci.bazelrc
+        env:
+          CI_BAZELRC_FILE_CONTENT: ${{ secrets.CI_BAZELRC_FILE_CONTENT }}
       - name: Lint
-        run: swift run swiftlint --reporter github-actions-logging --strict 2> /dev/null
+        run: |
+          export PATH="/usr/share/swift/usr/bin:$PATH"
```

Comment:
You'll need a Swift Docker container because Swift isn’t part of the standard Ubuntu Docker image.

Reply:
Swift 6.1.2 listed in their docu: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#language-and-runtime

With the Docker container, there's no Bazel.

Reply:
Ah got it these are the GHA images, not Docker.